### PR TITLE
[stable/prometheus-operator] Allow to configure kube-proxy endpoints

### PR DESCRIPTION
**Pull request by mcanevet**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by mcanevet** [comment]
@vsliouniaev @gianrubio conflict fixed

File: stable/prometheus-operator/templates/exporters/kube-proxy/endpoints.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.kubeProxy.enabled .Values.kubeProxy.endpoints }}
```
Comment (vsliouniaev) [inline stable/prometheus-operator/templates/exporters/kube-proxy/endpoints.yaml:1]: I think the `kube-scheduler` should be `kube-proxy`

**Review by vsliouniaev — CHANGES_REQUESTED** [review]

**Comment by vsliouniaev** [comment]
Could you please resolve the DCO issues? @mcanevet

**Pull request merged**